### ISSUE TITLE
build:prerender fails

### ISSUE THREAD

**Issue by mohammedzamakhan** [kamilmysliwiec/universal-nest#10]
When I ran 
`npm run build:prerender && npm run serve:prerender`

I got this error

```
> nest-nguniversal-demo@0.1.0 generate:prerender /Users/Zama/Desktop/passion/universal-nest
> cd dist && node prerender

/Users/Zama/Desktop/passion/universal-nest/dist/prerender.js:121605
'function' === typeof document.createEvent ? function CustomEvent (type, params) {
                      ^

ReferenceError: document is not defined
    at Object.<anonymous> (/Users/Zama/Desktop/passion/universal-nest/dist/prerender.js:121605:23)
    at __webpack_require__ (/Users/Zama/Desktop/passion/universal-nest/dist/prerender.js:20:30)
```

**Comment by hiepxanh** [comment]
me too

**Comment by kamilmysliwiec** [comment]
Fixed by #14 :)
